docker/nginx.dockerfile

```diff
@@ -0,0 +1,11 @@
+from nginx:latest
+
+RUN rm /etc/nginx/conf.d/*
+
+COPY nginx /etc/nginx
+
+RUN mkdir /etc/nginx/sites-enabled
+
+RUN cp /etc/nginx/sites-available/* /etc/nginx/sites-enabled/
```

Comment:
This should really be a symlink rather than a copy to ensure consistency between the files

Reply:
it's not symlinked because these files are not mounted anywhere as a volume, it's copied directly. I've had problems before with symlinks in docker file where it doesn't accept wildcards